packager/packager.cc

```diff
@@ -705,9 +717,61 @@ struct PackageInfoFinder {
             }
         }
 
+        if (send.fun == core::Names::sorbet() && send.numKwArgs() == 2) {
```

Comment:
Do we emit an error if you pass more or fewer args than 2? Should we?

Reply:
We do: that's handled by the type of the `sorbet` method, which has two require keyword arguments. If we don't pass either argument, or we pass too many, then we'll get those errors from the typechecking pass. That said, all valid invocations of this will have only two arguments, so if we don't have exactly two, then we know another error will get raised later on.